vllm/model_executor/layers/quantization/torchao.py

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: Apache-2.0
+import importlib.metadata
+from typing import Any, Dict, List, Optional
+
+import torch
+import torch.nn.functional as F
+from packaging import version
+from torch.nn.parameter import Parameter
+
+from vllm.model_executor.layers.linear import LinearBase, LinearMethodBase
+from vllm.model_executor.layers.quantization.base_config import (
+    QuantizationConfig)
+from vllm.model_executor.layers.quantization.torchao_utils import (
+    torchao_quantize_param_data)
+from vllm.model_executor.utils import set_weight_attrs
+
+
+class TorchAOConfig(QuantizationConfig):
+    """Config class for torchao."""
+    from torchao.core.config import AOBaseConfig
```

Comment:
Won't this throw an `ImportError` if torchao is not installed? `TorchAOConfig` is imported from `__init__` so this will always run unless I'm missing something

Reply:
you are right, will fix